drag_and_drop_v2/locale

```diff
@@ -0,0 +1 @@
+translations
```

Comment:
☝️

Reply:
@ahmedaljazzar We need this `locale/translations` symlink because this is [where edx-platform looks XBlock for translation files](https://github.com/edx/edx-platform/blob/2f12e0b4c47e99aaaf900b5288d33f902aba791d/common/lib/xmodule/xmodule/modulestore/django.py#L371).